src/components/PaginationWithPage.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import times from 'lodash.times';
+import DropdownSelect from './DropdownSelect';
+
+
+const defaultButton = props => (
+  <button type="button" className="send-request-btn btn btn-secondary active" {...props}>
+    {props.children}
+  </button>);
+
+const availablePageSize = [ 10, 25, 50];
+
+export default class PaginationWithPage extends React.Component {
+  constructor(props) {
+    super();
+    this.state = {
+      visiblePages: this.getVisiblePages(props.page, props.pages, props.screenWidth)
+    };
+  }
+
+  static propTypes = {
```

Comment:
1. propTypes contains no `screenWidth` property.
2. Rename `screenWidth` to `screenSize`.